### ISSUE TITLE
[Feature]: Add GNU-Screen and tmux indication at PS1

### ISSUE THREAD

**Issue by dr-costas** [Bash-it/bash-it#2185]
### Expected behavior

Have an indication of number of active GNU Screen or TMUX sessions at the PS1 when outside of GNU Screen or TMUX, and an indication of the name of the GNU Screen or TMUX session when inside GNU Screen or TMUX.

### Current behavior

There is no indication of the number of active GNU Screen or TMUX sessions in PS1 when outside of each of those, and when inside there is no indication of the name of the current session.

**Comment by dr-costas** [comment]
Hey! Thank you for making this into a feature request. I have edited it to reflect my initial thoughts.

**Comment by dr-costas** [comment]
Updated the title because I just now saw that the title was empty...

**Comment by seefood** [comment]
This is on a per-theme basis, sounds to me. I think it would need a function to figure out if it's inside a virtual term, and then have the themes display a conditional badge. If tmux or screen leave environment vars set, it's the easiest to detect. also look at oh-my-posh.

**Comment by akinomyoga** [comment]
tmux sets `TMUX`, and screen sets `STY` and `WINDOW`.

**Comment by seefood** [comment]
Yup, so Costas, feel free to make a pull request on any specific theme you want to add this, as there is very little code reuse between them. Personally I moved all my shells to oh-my-posh which supports a long list of such env indicators (sadly ble.sh breaks some of the graphic elements in it, but that's a separate issue for a different repo :)
